client/src/styles/fluent.scss

```diff
@@ -45,11 +48,21 @@ $fluent-color-inherited-text-badge: rgb(122, 69, 0);
       }
     }
 
+    .status-fluentfiltered {
+      a {
+        color: $fluent-color-filtered-text;
+      }
+    }
+
     span {
       &.badge {
         &.status-fluentinherited {
           color: $fluent-color-inherited-text-badge;
         }
+
+        &.status-fluentfiltered {
+          display: none;
```

Comment:
By default, I've hidden the flag text so that it doesn't conflict with the text from other flags (EG: "Draft", "Inherited", etc).

Potentially instead of adding a flag I could just append an extra class (somehow?), but I thought having the text there (in code) would mean other devs can just add their own style to display it (if they desire).

Reply:
Could it be shown on hover or something instead?

What does the "conflict" look like? maybe we could make it work / fit.

Reply:
@tractorcow I've changed the font colour just for the purposes of easily showing how it displays (otherwise, atm the default colour for this flag text is white):

- "About Us" is inheriting content and has not been added to "Filtered Locales".
- "Page not found" is drafted in the current Locale but has not been added to "Filtered Locales".
- "Server error" is published in the current Locale but has not been added to "Filtered Locales".

![screen shot 2017-11-21 at 11 54 02 am](https://user-images.githubusercontent.com/505788/33045758-cf035798-ceb2-11e7-851f-9c516e01d7bc.png)